code-for-canada/project-thundercat
# [Test Questions] Loading test questions from APIs on test start

# Description

This PR is introducing test questions content loading from the APIs. Now the questions (emails in our case) are loaded using the LoadTestContent actions.

Still have more work to do on future PRs to completely remove the local json file that contains all our test content.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Emails view - English:**

![image](https://user-images.githubusercontent.com/23021242/59941007-388fdb80-942a-11e9-9054-1370a92e7b3c.png)

**Emails view - French:**

![image](https://user-images.githubusercontent.com/23021242/59941021-4180ad00-942a-11e9-9d1f-0c8cbb2306e1.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB sample test.
2.  Make sure that the content is there and that you can switch language.

# Checklist
